fixtures/venmo.csv

```diff
@@ -0,0 +1,36 @@
+Username,ID,Datetime,Type,Status,Note,From,To,Amount (total),Amount (fee),Funding Source,Destination,Beginning Balance,Ending Balance,Statement Period Venmo Fees,Year to Date Venmo Fees,Disclaimer
```

Comment:
Did you mean to use this fixture in a test? It would be helpful to have a test to ensure that the CSV file is parsed correctly. There are examples in https://github.com/egh/ledger-autosync/blob/master/tests/test_converter.py

Reply:
I added it just so others could get a sense of the structure of the data.  I can push tests but they'll be incorrect/incomplete until there is a proper way to handle `assertions`, which may require work outside of the scope of the PR.

Reply:
OK, thank you! I'm planning to switch to pytest, but it may be some time.